### PR TITLE
Credit card payment page

### PR DESCRIPTION
**Pull request by KaiNissen**
add presenter for credit card payments
add generator for payment provider iframe target url

### CODE REVIEW

File: app/config/config.dist.json [hunk 1]
```diff
@@ -91,5 +91,12 @@
 		"return-url": "",
 		"cancel-url": "",
 		"item-name": ""
+	},
+	"creditcard": {
+		"base-url": "https://www.paypal.com/cgi-bin/webscr?",
```
Comment (gbirke) [inline app/config/config.dist.json]: Maybe something different or empty?

File: app/templates/CreditCardPayment.html.twig [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+					<strong>Ihre Daten sind sicher</strong>
+				</p>
+				<p>
+					Wir fühlen uns dem Datenschutz verpflichtet. Unsere Bestimmungen dazu können Sie
+					<a class="wlightbox inline" href="Datenschutz" data-href="#wlightbox-datenschutz">hier</a> einsehen
+				</p>
+			</div>
+		</div>
+	</div>
+	<div class="wlightbox-contents">
```
Comment (gbirke) [inline app/templates/CreditCardPayment.html.twig]: Please add TODO comment here to put the lightbox contents into separate templates.

File: tests/System/Routes/AddDonationRouteTest.php [hunk 3]
```diff
@@ -283,4 +283,33 @@ private function newValidPayPalInput() {
 		];
 	}
 
+	public function testGivenValidCreditCardData_redirectsToIframeEmbeddingPage() {
```
Comment (gbirke) [inline tests/System/Routes/AddDonationRouteTest.php]: Shouldn't that be `testGivenValidCreditCardData_showsIframeEmbeddingPage`? Or do we really have a redirection here?

**Comment by JeroenDeDauw** [comment]
Did a quick pass over this and did not see any issues, so +1 so far. Will do a more thorough review later today if this PR has not been merged then yet

**Comment by JeroenDeDauw** [comment]
Note that I did not review the template

**Pull request merged**